libs/wingsdk/src/target-sim/schema-resources.ts

```diff
@@ -59,8 +59,8 @@ export interface ApiAttributes {
 
 /** Schema for cloud.Api.props.routes */
 export interface ApiRoute {
-  /** The path to handle. */
-  readonly path: string;
+  /** The HTTP path pattern to handle. */
+  readonly pathPattern: string;
```

Comment:
I changed the name of this field to be more distinctive so in the snapshot sanitizer, I can add it as an exclusion to the absolute-path-removing rule